duet/dashboards/firefox_mr1_launch.dashboard.lookml

```diff
@@ -0,0 +1,2932 @@
+    sorts: [version_uplift.submission_date desc]
+    limit: 500
+    x_axis_gridlines: false
+    y_axis_gridlines: true
+    show_view_names: false
+    show_y_axis_labels: true
+    show_y_axis_ticks: true
+    y_axis_tick_density: default
+    y_axis_tick_density_custom: 5
+    show_x_axis_label: true
+    show_x_axis_ticks: true
+    y_axis_scale_mode: linear
+    x_axis_reversed: false
+    y_axis_reversed: false
+    plot_size_by_field: false
+    trellis: ''
+    stacking: percent
+    limit_displayed_rows: false
+    legend_position: center
+    point_style: circle
+    show_value_labels: false
+    label_density: 25
+    x_axis_scale: auto
+    y_axis_combined: true
+    ordering: none
+    show_null_labels: false
+    show_totals_labels: false
+    show_silhouette: false
+    totals_color: "#808080"
+    y_axes: [{label: '', orientation: left, series: [{axisId: version_uplift.uplift,
+            id: version_uplift.uplift, name: Uplift}], showLabels: true, showValues: true,
+        maxValue: 100, minValue: 0, valueFormat: '', unpinAxis: false, tickDensity: default,
+        tickDensityCustom: 5, type: linear}]
+    label_value_format: ''
+    series_types: {}
+    show_null_points: true
+    interpolation: linear
+    defaults_version: 1
+    listen: {}
+    row: 6
+    col: 0
+    width: 8
+    height: 8
+  - title: Desktop
+    name: Desktop
+    model: DUET
+    explore: version_uplift
+    type: single_value
+    fields: [version_uplift.submission_date, version_uplift.uplift]
+    fill_fields: [version_uplift.submission_date]
+    sorts: [version_uplift.submission_date desc]
+    limit: 500
+    custom_color_enabled: true
+    show_single_value_title: true
+    show_comparison: false
+    comparison_type: value
+    comparison_reverse_colors: false
+    show_comparison_label: true
+    enable_conditional_formatting: false
+    conditional_formatting_include_totals: false
+    conditional_formatting_include_nulls: false
+    value_format: "#.##"
+    conditional_formatting: [{type: equal to, value: !!null '', background_color: "#1A73E8",
+        font_color: !!null '', color_application: {collection_id: 7c56cc21-66e4-41c9-81ce-a60e1c3967b2,
+          palette_id: 56d0c358-10a0-4fd6-aa0b-b117bef527ab}, bold: false, italic: false,
+        strikethrough: false, fields: !!null ''}, {type: equal to, value: !!null '',
+        background_color: "#1A73E8", font_color: !!null '', color_application: {collection_id: 7c56cc21-66e4-41c9-81ce-a60e1c3967b2,
+          palette_id: 56d0c358-10a0-4fd6-aa0b-b117bef527ab}, bold: false, italic: false,
+        strikethrough: false, fields: !!null ''}]
+    series_types: {}
+    defaults_version: 1
+    hidden_fields: [version_uplift.submission_date]
+    listen: {}
+    row: 4
+    col: 0
+    width: 8
+    height: 2
+  - name: ''
+    type: text
+    title_text: ''
+    subtitle_text: ''
+    body_text: |-
+      -------------------------
+      # Upgrade Adoption
+
+      Release Date: 2021-06-01
```

Comment:
Should `# Upgrade Adoption` be the `title_text`? It looks like Looker doesn't renders the `body_text`, but instead just shows the raw text

Reply:
putting it in body text was by design ... so the markdown is rendering in some spots, but not in all of them,